src/services/fileManager.ts

```diff
@@ -49,43 +49,49 @@ import Logger from '../logger';
 const logger = Logger('FileManager');
 
 export async function getOrCreateFileFromRdcData(
-  rdpcFileDocument: RdpcFileDocument,
-  dataCenterId: string,
+	rdpcFileDocument: RdpcFileDocument,
+	dataCenterId: string,
 ): Promise<File> {
-  const fileToCreate: FileInput = {
-    analysisId: rdpcFileDocument.analysis.analysisId,
-    objectId: rdpcFileDocument.objectId,
-    programId: rdpcFileDocument.studyId,
-    repoId: dataCenterId,
+	const fileToCreate: FileInput = {
+		analysisId: rdpcFileDocument.analysis.analysisId,
+		objectId: rdpcFileDocument.objectId,
+		programId: rdpcFileDocument.studyId,
+		repoId: dataCenterId,
 
-    status: rdpcFileDocument.analysis.analysisState,
+		status: rdpcFileDocument.analysis.analysisState,
 
-    donorId: rdpcFileDocument.donors[0].donorId,
+		donorId: rdpcFileDocument.donors[0].donorId,
 
-    firstPublished: rdpcFileDocument.analysis.firstPublishedAt,
+		firstPublished: rdpcFileDocument.analysis.firstPublishedAt,
 
-    labels: [],
-  };
-  return await fileService.getOrCreateFileByObjId(fileToCreate);
+		labels: [],
+	};
+	return await fileService.getOrCreateFileByObjId(fileToCreate);
 }
 
-export async function updateStatusFromRdpcData(file: File, rdpcFileDocument: RdpcFileDocument): Promise<File> {
-  // Update relevant properties from RDPC
-  const status = rdpcFileDocument.analysis.analysisState;
-  if (status !== file.status) {
-    // Update the status of the object and in the DB
-    await fileService.updateFileSongPublishStatus(rdpcFileDocument.objectId, {
-      status: rdpcFileDocument.analysis.analysisState,
-    });
-    file.status = rdpcFileDocument.analysis.analysisState;
-  }
-
-  return file;
+/**
+ * Update File record from RDPC data and commit those changes to the file repo.
+ * @param file
+ * @param rdpcFileDocument
+ * @returns
+ */
+export async function updateFileFromRdpcData(file: File, rdpcFileDocument: RdpcFileDocument): Promise<File> {
```

Comment:
Renamed this function to better reflect its purpose. The `fileManager.ts` file was re-formatted to the updated prettier rules as a result.